charts/k8sosquery/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
I don't think this is needed.  You can remove the tests directory

Reply:
@gdandu-uptycs @hdsingh-uptycs If this is configured appropriately, post-installation of helm charts, users can run `helm test kubequery|k8sosquery` & it will execute all the helm tests. For example, it can confirm if the pod is ready to accept connections or not, etc.

This isn't mandatory but is good to have when there are a lot of resources involved in the helm chart & to verify the application status. Even the pod is running, due to TLS errors, etc. it might not be in a state to respond to incoming traffic. `helm test` helps us to detect them. It's equivalent to pod readiness probe but in helm context & is customizable.